src/bin/zmlicensectl

```diff
@@ -273,6 +273,7 @@ displayHelp()
 	echo "   --nalpeiron start|restart|stop|status"
 	echo "   --exportOfflineLicenseData | -exportOfflineLicenseData --exportStartTime YYYY/MM/DD --exportEndTime YYYY/MM/DD (offline only feature)"
 	echo "   --clearLicenseWorkDir"
+	echo "   --service getLogLevel"
```

Comment:
do we need to add **--service getOfflineMode** here

Reply:
done